src/local_package.ml

```diff
@@ -130,12 +130,20 @@ module Of_sctx = struct
       let ctx_build_dir = Path.as_in_build_dir_exn ctx.build_dir in
       Super_context.packages sctx
       |> Package.Name.Map.map ~f:(fun (pkg : Package.t) ->
+        let project_build_dir =
+          let dir = Path.Build.append_source ctx_build_dir pkg.path in
+          let scope = Super_context.find_scope_by_dir sctx dir in
+          Path.Build.append_source
+            dir
+            (Scope.project scope
+             |> Dune_project.root)
```

Comment:
Isn't this going to produce:

```
_build/default/<path-to-project>/<path-to-project>
```

?

More precisely, isn't `pkg.path` the project root already?

Reply:
Oh yeah. I changed it to use the package path. Will add a test before merging.